### PR TITLE
LPD-7203 Link notification to the Conflicts Checking page

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPD-7203](https://liferay.atlassian.net/browse/LPD-7203)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:test:stop

**Comment by gislayne-vitorino** [comment]
ci:stop

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/test/PublicationUserNotificationHandlerTest.java [hunk 1]
```diff
@@ -80,6 +84,11 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@After
+	public void tearDown() throws Exception {
+		ServiceContextThreadLocal.popServiceContext();
+	}
+
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-test/src/testIntegration/java/com/liferay/change/tracking/internal/test/PublicationUserNotificationHandlerTest.java:91]: If you're popping the service context in a tearDown, you should do the push in the setUp and assign the service context object to a class var. 

Another option is to do it locally in each tests with a try-finally bock.

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by pei-jung** [comment]
Good work, @gislayne-vitorino! See my changes in https://github.com/liferay-publications/liferay-portal/pull/1013.

**Pull request closed without merging**